lib/HTTP/Status.pm

```diff
@@ -56,14 +57,15 @@ my %StatusCode = (
     410 => 'Gone',
     411 => 'Length Required',
     412 => 'Precondition Failed',             # RFC 7232: Conditional Request
-    413 => 'Payload Too Large',
+    413 => 'Content Too Large',
     414 => 'URI Too Long',
     415 => 'Unsupported Media Type',
     416 => 'Range Not Satisfiable',           # RFC 7233: Range Requests
     417 => 'Expectation Failed',
-#   418 .. 420
+    418 => "I'm a teapot",                    # RFC 9110: Its official now -sorta
```

Comment:
Yes. However becauee of the HTTP_UNUSED being there twixe I opted for keeping the Teapot reference. Also because the RFC refers to this fact as the reason why it assigned.

Reply:
As much as it hurts my personal feeling that `I'm aTeapot` will be gone, I prefer correctness above sentiment.

Reply:
Not sure if we had Easter on 1-April in that year to0 ... butt yes, this is what I prefer, for those that ever used it to keep it alive.